testing/generate-typing-rewrite-info

```diff
@@ -15,6 +15,13 @@ else:
     from importlib_metadata import version
 
 
+# --- typing_extensions notes ---
+# - Starting with 3.11, NamedTuple and TypedDict can inherit from Generic
+CUSTOM_TYPING_EXT_SYMBOLS = {
+    (3, 11): {'NamedTuple', 'TypedDict'},
+}
```

Comment:
ah oh I mean I would keep the data here but just delete the 3.11 stuff from the output when copying it over (that way when 3.11 final comes out I can just rerun this)